layouts/About.tsx

```diff
@@ -17,10 +20,16 @@ const AboutLayout: FC<PropsWithChildren> = ({ children }) => (
         <main>{children}</main>
 
         <WithMetaBar />
+        <div className={styles.mobileFooter}>
```

Comment:
The article layout css adds border-top to metabar with nth-child(2) > last-child rule. It has to be modified to nth-child(2) > nth-child(2) to keep the existing behavior. With this change, metabar does not have a border and the footer has a thick border.

Reply:
I have resolved the issue and it's working fine, will be  commiting that code.
Please look into it for further issue.